### ISSUE TITLE
User creation workflow is broken

### ISSUE THREAD

**Issue by elv-arun** [eluv-io/elv-fabric-browser#13]
Epic: qluvio/info#362

User creation workflow is broken. Last few times I tried, what happened to me was: I created the user and then I fund it. When I get back to name it, it has disappeared from the browser store. So it requires having copied the private key and needs to be re-created to save it in browser store.
```
Accounts with zero balance do not seem to get saved to browser local storage when added. (both when generating a new account and when adding an existing zero-balance account to browser)
Replication
Go to https://core.v3.contentfabric.io/ then Add Account, Mnemonic, Generate Mnemonic, enter new password, submit. Account shows in list. Copy account address.
In new browser window, log in with different account, transfer funds to first address (of new user).
Reload original browser window
New account not listed. If user forgot to save mnemonic, the funds are inaccessible.
Expected behavior -  persist across browser page refresh, and once account is funded it would be possible to enter password chosen in step 1 and access without having to re-add account to browser. (edited) 
9:35
Same if 2 different browsers are used for steps (e.g. Safari to create, Chrome to fund)
```

**Comment by elv-kevin** [comment]
(deployed on demo)

**Comment by elv-arun** [comment]
AA to verify.

**Comment by elv-gerald** [comment]
Verfied on demov3 frowser
I created a new account with a mnemonic 
1. the account had 0 balance
2. I refreshed the page and verified the account was still present
3. I hard reloaded the page and verified the account was still present
4. I added the 0 balance account on abother browser that did not have it and verified the account was not removed after reloaded.

Tested withe chrome and firefox.

![image](https://user-images.githubusercontent.com/79117605/201748922-f39e8719-1554-46de-a601-5e4db4ad8f01.png)
